dbver.py

```diff
@@ -0,0 +1,77 @@
+import sqlalchemy, os, sys, config
+from migrate.versioning import api as versioning_api
+from macpath import curdir
+from gtk.keysyms import currency
+
+_ENGINE = None
+
+def get_engine():
+    global _ENGINE
+    if _ENGINE:
+        return _ENGINE
+    
+    sql_connection = config.App.db_uri(config)
+    engine_args = {
+                    "pool_recycle": 3600,
+                    "echo": False
+                    }
+  
+    _ENGINE = sqlalchemy.create_engine(sql_connection, **engine_args)
+  
+    _ENGINE.connect()
+    return _ENGINE
+  
+def get_repository():
+    return sys.path[0] + os.sep + "dbver"
+  
+def db_version():
+    repository = get_repository()
+    try:
+        return versioning_api.db_version(get_engine(), repository)
+    except :
+        # if we aren't version controlled we may already have the database
+        # in the state from before we started version control, check for that
+        # and set up version_control appropriately
+        meta = sqlalchemy.MetaData()
+        engine = get_engine()
+        meta.bind = engine
+        tables = meta.tables
+        if len(tables) == 0:
+            db_version_control(000)
+            return versioning_api.db_version(get_engine(), repository)
+        else:
+            # db has tables without migrate_version, it out of control
+            raise
```

Comment:
According to [PEP8](https://www.python.org/dev/peps/pep-0008/#blank-lines) there should be blank lines after top level function definitions.